README.md

```diff
@@ -135,7 +135,12 @@ Use the following commands:
 
 ```shell
 $ docker-compose build ubuntu-cpp
-$ docker-compose run ubuntu-cpp
+$ docker-compose run --rm ubuntu-cpp
+```
+If the compilation uses too many resources, you can limit the number
+of threads deployed in the container by doing:
```

Comment:
Maybe only a final nit I have at this point is to update this comment to reflect not only overriding this in case of using too many resources but in case of wanting to increase the number of threads as some users might want to do that too. I am thinking on something like:
```
If you want to increase or decrease the number of threads used when building Velox you can override the `NUM_THREADS` environment variable by doing:
```